qiita_ware/ebi.py

```diff
@@ -328,6 +316,7 @@ def generate_sample_xml(self):
                                              "/sra_1_3/SRA.sample.xsd"})
 
         for sample_name, sample_info in sorted(viewitems(self.samples)):
+            sample_info = dict(sample_info)
```

Comment:
I'm curious here - what is the original type of sample_info and why are you casting it here to a dict?